packages/dev/docs/pages/react-aria/home/ExampleApp.tsx

```diff
@@ -245,6 +245,16 @@ export function ExampleApp() {
                     <MenuItem id="favorite"><StarIcon aria-hidden className="w-4 h-4" /> {item.isFavorite ? 'Unfavorite' : 'Favorite'}</MenuItem>
                     <MenuItem id="edit"><PencilIcon aria-hidden className="w-4 h-4" /> Edit…</MenuItem>
                     <MenuItem id="delete"><TrashIcon aria-hidden className="w-4 h-4" /> Delete…</MenuItem>
+                    <SubmenuTrigger>
```

Comment:
Can you create a followup task to add a header and section to this example some time?